src/index.ts

```diff
@@ -166,6 +166,7 @@ export { default as useCloseDropDown } from "./hooks/useCloseDropDown";
 // export { default as useMultiSelect } from "./hooks/useMultiSelect";
 export { default as useNYPLBreakpoints } from "./hooks/useNYPLBreakpoints";
 export { default as useNYPLTheme } from "./hooks/useNYPLTheme";
+export { default as useScrollTabIntoView } from "./hooks/useScrollTabIntoView";
```

Comment:
Ah but here it is being exported. I think it could be generalized if the query text is an argument and `"[role=tab][aria-selected=true]"` is the default value. Otherwise, this might be hard to use outside of the specific DS `Tabs` component.

Reply:
Removed this since it is not ideal to create/pass a parameter for the dependency array used in the useEffect hook.